### PR TITLE
add http-params-serializable

### PR DESCRIPTION
**Pull request by dukenguyenxyz**

### CODE REVIEW

File: src/openapi-generator/helpers/action-controller.cr [hunk 1]
```diff
@@ -428,4 +429,28 @@ module OpenAPI::Generator::Helpers::ActionController
...
+
+      private def get_params : Params
+        # QP_LIST.each{|k,v|
+        #   pp! k
+        #   pp! v.first
+        # }
+        Params.from_query(request.query.not_nil!)
+      end
+    end
+  end
```
Comment (dukenguyenxyz) [inline src/openapi-generator/helpers/action-controller.cr:455]: @caspiano not exactly sure why this is not working

Comment (dukenguyenxyz) [reply src/openapi-generator/helpers/action-controller.cr:455]: it compiles. when I pp! items from QP_LIST in private def get_params they do get output, but apparently none get declared as instance_var in the struct when looping over.

File: shard.yml [hunk 2]
```diff
@@ -14,6 +14,9 @@ dependencies:
   open_api:
     github: elbywan/open_api.cr
     commit: 7ad39e980ff64dfa88414ca0d2c8515159ab51ae
+  http-params-serializable:
+    github: vladfaust/http-params-serializable
+    version: ~> 0.4.0
```
Comment (caspiano) [inline shard.yml:19]: Could use a looser pin. Also my fork has a bunch of updates that I've made

Comment (dukenguyenxyz) [reply shard.yml:19]: https://github.com/caspiano/http-params-serializable
=> `This branch is even with vladfaust:master`

Comment (caspiano) [reply shard.yml:19]: See this branch
https://github.com/caspiano/http-params-serializable/tree/chore/0.36.0

**Comment by dukenguyenxyz** [comment]
This issue is resolved on `feat/action-controller`

**Pull request closed without merging**